models/index.js

```diff
@@ -23,7 +23,8 @@ fs
   });
 
 Object.keys(db).forEach(function(modelName) {
-  if ("associate" in db[modelName]) {
+  //if ("associate" in db[modelName]) { // 
+  if (db[modelName].associate) { 
```

Comment:
associate is not required attribute for model definition 
which happened if model attribute associate doesn't have a type function?